packages/firebase_remote_config/firebase_remote_config/lib/src/firebase_remote_config.dart

```diff
@@ -37,9 +37,7 @@ class FirebaseRemoteConfig extends FirebasePluginPlatform with ChangeNotifier {
   /// Returns an instance using the specified [FirebaseApp].
   static FirebaseRemoteConfig instanceFor({required FirebaseApp app}) {
     return _firebaseRemoteConfigInstances.putIfAbsent(app.name, () {
-      final instance = FirebaseRemoteConfig._(app: app);
-      FirebasePluginPlatform.verify(instance);
-      return instance;
+      return FirebaseRemoteConfig._(app: app);
```

Comment:
How come this was removed?

Reply:
It wasn't here before this PR. It was added in a previous commit and removed in this one.
Since we cannot `set` the instance of FirebaseRemoteConfig, there is no need to add a verify right here